mainnet/24395.json

```diff
@@ -0,0 +1 @@
+[{"message":{"validator_index":"24395","from_bls_pubkey":"0xa7f81c1480893a04f57018f6fde7c4abc6305933c7cd0f989925ccf8ea0e4c3d4b331aea5437d292b85104bddf4c7161","to_execution_address":"0x3ba9c1788ffd62574a117d16d3d3e9dc4e3053d8"},"signature":"0x9804c3e7b39d1dc2482726555adfad41790490d7effa02193ba6999707765e742a4850c4294a50430f2e7c6273897ac60340264f6699c38d48e272e4c678168f441e565d1eef1f9fe4dc4d10984206ced01c6b217f936633a242f5a33fd6e1c4"}
```

Comment:
Looks like a missing ] at the end of this file. 
Mind modifying this file and committing it? We can then re-run the checks and it likely will pass.

Reply:
Oops, thanks.